### PR TITLE
Addres some more of the errors we were having

### PR DESCRIPTION
**Pull request by fletcherdss**

### CODE REVIEW

File: src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java [hunk 1]
```diff
@@ -2657,20 +2657,29 @@ public String cleanTitle(String title) {
...
             return null;
         } catch(IllegalArgumentException ex){
             // Handles case: java.lang.IllegalArgumentException: Not a valid domain name: '221.214.182.123'
             return null;
         } catch(java.lang.IllegalStateException ex){
             // Handles case: java.lang.IllegalStateException: Not under a public suffix: developer.team
+	    logger.info(ex.toString());
             return null;
-        }
+        } catch(java.lang.IllegalArgumentException ex){ //happens when url is: http://<IP address>
```
Comment (abhishekvm) [inline src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java:2678]: @fletcherdss @skyshard Multiple catch blocks for IllegalArgumentException - compiler error

Comment (fletcherdss) [reply src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java:2678]: Looks like it was from a bad merge, not sure how it slipped through. Thanks for catching that.

Comment (fletcherdss) [reply src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java:2678]: Oh I see, it looks like a pull request from a while ago got merged and a duplicate catch block had been added a while ago.

Comment (abhishekvm) [reply src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java:2678]: Created patch to fix it https://github.com/skyshard/snacktory/pull/20

**Pull request merged**